pkg/keyspace/tso_keyspace_group.go

```diff
@@ -71,15 +76,8 @@ func (m *GroupManager) Bootstrap() error {
 		return err
 	}
 
-	userKind := endpoint.StringUserKind(defaultKeyspaceGroup.UserKind)
```

Comment:
delete line 74 ~ 79 for two reasons:
1. This is covered by the for loop below in which it loads all keyspace groups, including the default keyspace group, and initialize m.groups.
2. Theoretically,  at line 79, we put defaultKeyspaceGroup into m.groups, but defaultKeyspaceGroup might be different from the latest version persisted in the store, though later on the current logic of indexedHeap will overwrite the existing one if put a new one.

Reply:
@rleungx ^

Reply:
Let me know if I missed anything.

Reply:
You're right.